assets/templates/partials/json-ld/dataset/filterable.tmpl

```diff
@@ -1,15 +1,17 @@
 {{ $data := .DatasetLandingPage }}
+{{ $URLPath := .URI }}
 {{ $length := len $data.Version.Downloads }}
+{{ $SiteDomain := .SiteDomain }}
 {{ if gt $length 0 }}
         "distribution": [
         {{ range $i, $el := $data.Version.Downloads }} {
             "@type": "DataDownload",
             "encodingFormat": {{ .Extension }},
-            "contentUrl": {{ .URI }}
+            "contentUrl": {{concatenateStrings $SiteDomain .URI }}
```

Comment:
I think we should probably add the `Subdomain` logic to this even though we don't support welsh here yet.  It will save us coming back when we do.

Reply:
Fair point; have updated